client/asset/eth/nodeclient_harness_test.go

```diff
@@ -2167,31 +2270,37 @@ func TestReplayAttack(t *testing.T) {
 		// Set some variables in the contract used for the exploit. This
 		// will fail (silently) due to require(msg.origin == msg.sender)
 		// in the real contract.
-		txOpts, _ := cl.txOpts(ctx, 1, defaultSendGasLimit*5, nil)
-		_, err := reentryContract.SetUsUpTheBomb(txOpts, ethSwapContractAddr, secretHash, big.NewInt(inLocktime), participantAddr)
+		txOpts, err := cl.txOpts(ctx, 1, defaultSendGasLimit*5, nil, nil)
+		if err != nil {
+			t.Fatalf("txOpts error: %v", err)
+		}
+		_, err = reentryContract.SetUsUpTheBomb(txOpts, ethSwapContractAddr, secretHash, big.NewInt(inLocktime), participantAddr)
 		if err != nil {
 			t.Fatalf("unable to set up the bomb: %v", err)
 		}
-		if err = waitForMined(cl, 10, false); err != nil {
+		if err = waitForMined(10, false); err != nil {
 			t.Fatal(err)
 		}
 	}
 
-	txOpts, _ = cl.txOpts(ctx, 1, defaultSendGasLimit*5, nil)
+	txOpts, err = cl.txOpts(ctx, 1, defaultSendGasLimit*5, nil, nil)
+	if err != nil {
+		t.Fatalf("txOpts error: %v", err)
+	}
 	txOpts.Value = nil
 	// Siphon funds into the contract.
 	tx, err := reentryContract.AllYourBase(txOpts)
 	if err != nil {
 		t.Fatalf("unable to get all your base: %v", err)
 	}
 	spew.Dump(tx)
-	if err = waitForMined(ethClient, 10, false); err != nil {
+	if err = waitForMined(10, false); err != nil {
 		t.Fatal(err)
 	}
 	receipt, err := waitForReceipt(t, ethClient, tx)
```

Comment:
This error is ignored.   However, it appears that the `t *testing.T` arg of `waitForReceipt` is unused.  Kinda looks like the intent was to have `waitForReceipt` do `t.Fatal`, but we went with an error return instead.  Probably easiest is to remove the input arg and use the error return everywhere.

Reply:
Removed the `*testing.T` argument and I think fixed the ignoring location.